src/Components/Components/src/Binding/CascadingModelBinder.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection.Metadata;
+using Microsoft.AspNetCore.Components.Binding;
+
+namespace Microsoft.AspNetCore.Components;
+
+/// <summary>
+/// Defines the binding context for data bound from external sources.
+/// </summary>
+public class CascadingModelBinder : IComponent
+{
+    private RenderHandle _handle;
+    private bool _hasRendered;
+    private ModelBindingContext? _bindingContext;
+
+    /// <summary>
+    /// The binding context name.
+    /// </summary>
+    [Parameter] public string Name { get; set; } = default!;
+
+    /// <summary>
+    /// The binding context name.
+    /// </summary>
+    [Parameter] public string BindingContextId { get; set; } = default!;
```

Comment:
What's the difference between `Name` and `BindingContextId`? Developers might struggle to understand this as they sound like the same thing (also: the XML docs are duplicated).

Is there possibly any other name that could differentiate these more clearly?